app/src/main/java/com/example/sabonit/UserLoginActivity.java

```diff
@@ -119,6 +121,7 @@ public void onComplete(@NonNull Task<DocumentSnapshot> task) {
                         Log.d("SearchAccountIdFailed", "No such document");
                         writeNewAccount();
                     }
+                    displayMessageIfCartIsEmpty();
```

Comment:
Great, now the user is informed if its cart is not empty from previous entries

Reply:
yes like what we talked about